kitsune/messages/signals.py

```diff
@@ -1,4 +1,4 @@
 import django.dispatch
 
 
-message_sent = django.dispatch.Signal(providing_args=["to", "text", "msg_sender"])
+message_sent = django.dispatch.Signal(["to", "text", "msg_sender"])
```

Comment:
I believe the signature should just be `Signal()`.

Copying from the Django 3.1 deprecation notice:
```
The purely documentational providing_args argument for [Signal](https://docs.djangoproject.com/en/4.1/topics/signals/#django.dispatch.Signal) is deprecated. If you rely on this argument as documentation, you can move the text to a code comment or docstring.
```

Reply:
Got it.